packages/terra-site/webpack.config.js

```diff
@@ -21,34 +30,42 @@ module.exports = {
     rules: [{
       test: /\.(jsx|js)$/,
       exclude: /node_modules/,
-      use: 'babel-loader',
+      use: [
+        !process.env.CI && threadLoaderRule,
```

Comment:
it seems to hang on travis, i'm wondering if this is because too many threads/processes are spawned.